iaas/20230823.md

```diff
@@ -0,0 +1,57 @@
+- [#269](https://github.com/SovereignCloudStack/issues/issues/269) - dito - (AI @horazont: organize touchpoint of Heiko with @fkr, Mon 11:05 CEST)
+- [#267](https://github.com/SovereignCloudStack/standards/issues/267) - review of PR [#319](https://github.com/SovereignCloudStack/standards/pull/319), more reviewers wanted, especially CSPs
+  - Possibly followup @fkr, @mbuechse, @garloff next week (for clarifying feedback, resolving, ...)
+- [#317](https://github.com/SovereignCloudStack/standards/issues/317) - collect things that should end up in v4 Standard, should get this mostly complete in Sprint Jena (starting today)
+  - prioritize this
+- [#265](https://github.com/SovereignCloudStack/standards/issues/265), PR [#323](https://github.com/SovereignCloudStack/standards/pull/323) - also work on this during Sprint Jena
+- [#326](https://github.com/SovereignCloudStack/standards/issues/326) - image names, source URLs (also target v4 standard/R5), source URLs not standardized well b/w upstream, image-manager URLs use mirror (minio.osism), new ticket, PR [#314](https://github.com/SovereignCloudStack/standards/pull/314)
+  - Challenge: ensure source URLs are stable (archived), otherwise the tracability may end after a day :-( -- but still useful to have "reproducability" for a limited amount of time
+  - Prefer versioned URLs
+- [osism-image-manger/#634](https://github.com/osism/openstack-image-manager/issues/634)
+  - Trouble: original URL is lost, as osism minio is used to grab images, *needs fixing*
+  - Is relevant, even if image-manager might not be extremely popular for historical reasons, as we want to have a complete reference implementation that deliver (ideally) all that's needed for a SCS-compatible IaaS
+  - In use on CiaB systems, C&H, wave ...
+  - plusserver crawler vs github action to update list of images
+    - Idea: SCS service with current image URL for providers?
+  - Unrelated: build dates need to be available in correct format, see <https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0102-v1-image-metadata.md#image-build-info>
+- Ceph stories: [#565](https://github.com/osism/issues/issues/565), [#595](https://github.com/osism/issues/issues/595), [#413](https://github.com/osism/issues/issues/413) -- vacation without delegation :-(
+
+### Add to sprint
+
+- *Run tests for SCS-compatible IaaS on Zuul* [standards/#316](https://github.com/SovereignCloudStack/standards/issues/316)
+  - Want to bring this into production use in SCS
+  - zuul infra needs modern IaC approach (redeployable any time), availability monitored, data backed up
+  - Move this forward! (Tracked in Epic [#157](https://github.com/SovereignCloudStack/issues/issues/157))
+- *Extend Nova API to support filters*, see [this comment](https://github.com/SovereignCloudStack/standards/issues/228#issuecomment-1680471609) in [standards/#228](https://github.com/SovereignCloudStack/standards/issues/228), but create new issue for this
```

Comment:
@fkr this was also not discussed, and it was part of my original comment, but you missed it

Reply:
well we certainly did discuss the zuul infra iac and monitoring stuff. I remember because I brought that topic up before and was asked about it at least twice during the meeting. :slightly_smiling_face:

Reply:
Don't be misled by the context that github shows. I only referred to the bottom-most line.